Sources/SwiftDocC/Model/Rendering/Content/RenderBlockContent.swift

```diff
@@ -60,6 +60,8 @@ public enum RenderBlockContent: Equatable {
     case termList(TermList)
     /// A table that contains a list of row data.
     case table(Table)
+    
+    case row(Row)
```

Comment:
Looks like we're missing docs here, same for the public APIs below

Reply:
Thanks! Addressed here: https://github.com/apple/swift-docc/pull/378/commits/d0c2e31d47fd60217994704c71566e13ea48744e.